### ISSUE TITLE
Dashboard for other databases then telegraf

### ISSUE THREAD

**Issue by core-process** [influxdata/chronograf#968]
I tried to setup a dashboard as follows:

```
curl -X POST -H "Content-Type: application/json" -d '{
    "cells": [
        {
            "x": 0,
            "y": 0,
            "w": 12,
            "h": 12,
            "name": "Arexx",
            "queries": [
                {
                    "query": "SELECT mean(\"temp\") FROM temp",
                    "db": "test",
                    "rp": "autogen",
                    "label": "°C",
                    "range": {
                        "upper": 80,
                        "lower": 0
                    },
                    "wheres": ["\"anlage\" = '10000011'"],
                    "groupbys": ["\"sensor\""]
                }
            ],
            "type": "line"
        }
    ],
    "name": "10000011"
}' "http://localhost:8888/chronograf/v1/dashboards"
```

**Note:** `"db": "test"`

Still the request issued by the UI to the route `/chronograf/v1/sources/1/proxy` references the `telegraph` database:

```
db: "telegraf",
query: "SELECT mean(\"temp\") FROM temp where time > now() - 15m and \"anlage\" = 10000011 group by time(1m),\"sensor\"",
rp: "autogen"
```

Bug or feature? :-)

**Comment by goller** [comment]
@core-process Thanks for writing in! Doh, bug for sure. We'll take a peek and figure it out.

Our goal is to have dashboard editing done soon so you won't have to go through POSTing, but, thanks for being brave :)

**Comment by goller** [comment]
@core-process I found the bug here: https://github.com/influxdata/chronograf/blob/master/ui/src/dashboards/components/Dashboard.js#L36

Fix incoming!

**Comment by core-process** [comment]
Thanks for your response! Ahh.. I found this line too, but wasn't sure, since I read the source code the first time ;-)

**Comment by goller** [comment]
@core-process Ok, fix has been tested and added to master in #969 .  Each time master builds it'll create a new docker container at `quay.io/influxdb/chronograf:latest`.  If you aren't using docker, you try to a build, or wait until Friday when we have our next release.

Thanks again for reporting it!

**Comment by core-process** [comment]
Thanks! :-)
